config/qualifications.yml

```diff
@@ -6,8 +6,7 @@ subtasks:
 ---
 title: 'Join UMTS on Github'
 description: >
-  Ask one of your supervisors or team leaders to get you added to
-  the relevant groups in our Github organization.
+  Make sure you have access to the relevant groups in our Github organization.
```

Comment:
No, the old wording was better. What are the "relevant groups". Note also that an organization can have private groups — those that you can't see if you're not a member of them (or an admin).

Reply:
I didn't put the specific groups on here because 1) The supervisor should definitely know and 2) it could change. But, if that's a bad reason, I'll just put the actual groups.

Reply:
For my own reference tomorrow: Going to change this to be something entirely different, I don't remember what I intended anymore but it was something about Github.